timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -209,39 +218,48 @@ def fit(
             used to compute the validation scores. Note that only the last ``prediction_length`` time steps of each
             time series are used for computing the validation score.
 
-            If ``None``, AutoGluon will split :attr:`train_data` into training and tuning subsets using
+            If None, AutoGluon will split :attr:`train_data` into training and tuning subsets using
```

Comment:
out of curiosity, why remove the backticks? we are referring to the keyword None after all.

Reply:
My bad, fixed this.